frontend/src/components/common/Footer.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  Container,
+  Flex,
+  Link,
+  Text,
+  useMediaQuery,
+  VStack,
+} from "@chakra-ui/react";
+import React from "react";
+
+import { Facebook, Instagram } from "./icons";
+
+const Footer = (): JSX.Element => {
+  const [isScreenLargerThan] = useMediaQuery("(min-width: 320px)");
+  return (
+    <Container pt="24px">
+      <VStack>
+        {isScreenLargerThan ? (
+          <>
+            <Flex>
+              <VStack pr="100px">
+                <Container>
+                  <Text
+                    marginBottom="24px"
+                    textStyle="mobileHeader1"
+                    color="raddish.100"
+                  >
+                    Community Fridge
+                  </Text>
+                  <Text>
+                    The Kitchener Market: <br />
+                    300 King Street East, <br />
+                    Kitchener, ON N2H 2V5 <br />
+                    (647) 607-1312 <br />
+                    communityfridgekw@gmail.com
+                  </Text>
+                </Container>
+              </VStack>
+              <Container>
+                <Text textStyle="mobileHeader2">Stay Updated!</Text>
+                <br />
+                <Link
+                  href="https://www.facebook.com/CommunityFridgeKW/"
+                  pr="18px"
+                >
+                  <Facebook />
+                </Link>
+                <Link href="https://www.instagram.com/communityfridgekw/">
+                  <Instagram />
+                </Link>
+              </Container>
+            </Flex>
+            <Text pt="24px">© Community Fridge KW 2021</Text>
+          </>
+        ) : (
+          <>
+            <Container>
+              <Text
+                marginBottom="24px"
+                textStyle="mobileHeader1"
+                color="raddish.100"
+              >
+                Community Fridge
+              </Text>
+              <Text>
+                The Kitchener Market: <br />
+                300 King Street East, <br />
+                Kitchener, ON N2H 2V5 <br />
+                (647) 607-1312 <br />
+                communityfridgekw@gmail.com
+              </Text>
+            </Container>
+            <Container>
+              <Text pt="18px" textStyle="mobileHeader2">
+                Stay Updated!
+              </Text>
+            </Container>
+            <Container>
+              <Link
+                href="https://www.facebook.com/CommunityFridgeKW/"
+                pr="18px"
+              >
+                <Facebook />
+              </Link>
+              <Link href="https://www.instagram.com/communityfridgekw/">
+                <Instagram />
+              </Link>
+            </Container>
+            <Container>
+              <Text pt="8px">© Community Fridge KW 2021</Text>
+            </Container>
+          </>
+        )}
+      </VStack>
+    </Container>
+  );
+};
```

Comment:
Instead of the ternary, we can do something like this for responsive styling: 
Essentially everywhere that you wrap with a `Container` should not also have `Flex` or `VStack` since they both extend from a `div` and so we can pass in the css props we need directly for [responsiveness](https://chakra-ui.com/docs/features/responsive-styles)

```tsx]
 <Container p="1.5rem">
      <Box display={{ md: "flex" }}>
        <VStack flexShrink={0} width={{ md: 40 }} align={{ md: "left" }}>
          <Text
            marginBottom="24px"
            textStyle="mobileHeader1"
            color="raddish.100"
          >
            Community Fridge
          </Text>
          <Text>
            The Kitchener Market: <br />
            300 King Street East, <br />
            Kitchener, ON N2H 2V5 <br />
            (647) 607-1312 <br />
            communityfridgekw@gmail.com
          </Text>
        </VStack>

        <VStack
          mt={{ base: 4, md: 0 }}
          align={{ md: "left" }}
          ml={{ base: "40%", lg: "40%", md: "0%", sm: "0%" }}
        >
          <Text textStyle="mobileHeader2" mb="12px">
            Stay Updated!
          </Text>
          <Box>
            <Link href="https://www.facebook.com/CommunityFridgeKW/" pr="18px">
              <Facebook />
            </Link>
            <Link href="https://www.instagram.com/communityfridgekw/">
              <Instagram />
            </Link>
          </Box>
        </VStack>
      </Box>
      <Text align={{ lg: "center" }} pt="24px">
        © Community Fridge KW 2021
      </Text>
    </Container>
```